### PR TITLE
Spells complete

### PR DESCRIPTION
**Pull request by dazdya**
Heya,
Spells are complete, or at least the ones from the player's handbook.

I did have one more issue. For the demona, the diablo and the wish spells, I used a list, but I couldn't exactly remember how I used to do that. I think I used <ul> and <li\> correctly, but it might be a good idea to check.

### CODE REVIEW

File: dnd/config/spells.yaml [hunk 1]
```diff
@@ -0,0 +1,1306 @@
...
+range: "short"
+area: "up to 6 demons"
+casting_time: "1 round"
+components: "VSM"
+duration: 1 round / level
+mechanics: !!null
+description: |
+  <p>This spell summons 1d3+1 demona from another reality to attack any targets the caster orders them to. The demona disappear when the duration ends or if they fall below 1 hp.</p>
+  <p>The demona are all of the same randomly generated type, with most their statistics decided by dice-rolls:<ul>
+  <li\>Their AC is 8 + 1d10 + the caster's circle.
```
Comment (nihlaeth) [inline dnd/config/spells.yaml]: the correct syntax for an unordered list is:
```html
<ul>
<li>list item one</li>
<li>list item two</li>
</ul>
```

File: dnd/config/spells.yaml [hunk 2]
```diff
@@ -0,0 +1,1306 @@
...
+  <p>This spell changes the target's race into its fivefold version, as described in chapter 2. If the target is affected, he can no longer be a player character. The spell does not affect trumen. It is a curse.</p>
+---
+name: Flame
+sphere: war
+circle: 1
+range: "caster"
+area: "2 meters"
+casting_time: "3"
+components: "VSM"
+duration:	instantaneous
```
Comment (nihlaeth) [inline dnd/config/spells.yaml]: any reason for the extra whitespace here?

File: dnd/config/spells.yaml [hunk 3]
```diff
@@ -0,0 +1,1306 @@
...
+---
+name: Gorgon
+sphere: thought
+circle: 1
+range: "short"
+area: "1 creature"
+casting_time: "3"
+components: "VSM"
+duration:	5 rounds or until discharged
+mechanics:	will negates, twice
```
Comment (nihlaeth) [inline dnd/config/spells.yaml]: Again, the extra whitespace. It shouldn't break anything, but I prefer conformity.

**Review by nihlaeth — CHANGES_REQUESTED** [review]
Looks awesome! The only two issues I have with this, are the list syntax and some extra whitespace here and there. Could you fix that?

**Comment by nihlaeth** [comment]
also, you can find information on styling here: https://www.w3schools.com/bootstrap/bootstrap_tables.asp

**Comment by nihlaeth** [comment]
Question for the code backend: do you want spells sorted on sphere or on circle? Does sphere even have significance for spells?

**Review by nihlaeth — APPROVED** [review]

**Comment by nihlaeth** [comment]
there are a few stray tabs left, but I'll fix those in my own editor

**Comment by dazdya** [comment]
The spells should be sorted on circle, high to low, and then alphabetically. The order in the character sheet is now correct.

However, I noticed that the text doesn't appear for spells that have more than one word in the name. Can this be fixed by putting "" marks around the title?

**Pull request merged**